app/controllers/candidate_interface/gcse/english/grade_controller.rb

```diff
@@ -76,7 +76,7 @@ def english_details_params
     end
 
     def view_path(update_path: false)
-      if gcse_qualification? && application_not_submitted_yet?
```

Comment:
This is not needed anymore on continuous applications world, so safely to delete it.